robust-checks.tex

```diff
@@ -1,134 +1,29 @@
-
-% color can be used to apply background shading to table cells only
-\usepackage[table]{xcolor}
-
-% array package and thick rules for tables
-\usepackage{array}
-
-% enumerate package lets us use letters instead of numbers
-\usepackage{enumerate}
-
-% create "+" rule type for thick vertical lines
-\newcolumntype{+}{!{\vrule width 2pt}}
-
-% create \thickcline for thick horizontal lines of variable length
-\newlength\savedwidth
-\newcommand\thickcline[1]{%
-  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
-  \cline{#1}%
-  \noalign{\vskip\arrayrulewidth}%
-  \noalign{\global\arrayrulewidth\savedwidth}%
-}
-
-% \thickhline command for thick horizontal lines that span the table
-\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
-\hline
-\noalign{\global\arrayrulewidth\savedwidth}}
-
-
 % Remove comment for double spacing
-%\usepackage{setspace}
-%\doublespacing
-
-% Text layout
-\raggedright
-\setlength{\parindent}{0.5cm}
-\textwidth 5.25in
-\textheight 8.75in
-
-% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
-% Captions will be left justified
-\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
-\renewcommand{\figurename}{Fig}
-
-% Use the PLoS provided BiBTeX style
-\bibliographystyle{plos2015}
-
-% Remove brackets from numbering in List of References
-\makeatletter
-\renewcommand{\@biblabel}[1]{\quad#1.}
-\makeatother
-
-% Leave date blank
-\date{}
-
-% Header and Footer with logo
-\usepackage{lastpage,fancyhdr,graphicx}
-\usepackage{epstopdf}
-\pagestyle{myheadings}
-\pagestyle{fancy}
-\fancyhf{}
-\setlength{\headheight}{27.023pt}
-\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
-\rfoot{\thepage/\pageref{LastPage}}
-\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
-\fancyheadoffset[L]{2.25in}
-\fancyfootoffset[L]{2.25in}
-\lfoot{\sf PLOS}
-
-%% Include all macros below
-\newcommand{\fixme}[2]{\textsc{\textbf{{#1}: {#2}}}}
-\newcommand{\recommend}[1]{\textit{#1}}
-\newcommand{\withurl}[2]{{#1}\footnote{\texttt{#2}}}
+\usepackage{setspace}
+\onehalfspacing
 
 \usepackage[ampersand]{easylist}
 
 \begin{document}
-\vspace*{0.2in}
 
 
 \begin{flushleft}
 {\Large
 \textbf\newline{Ten Simple Rules for Making Research Software More Robust: The Checklist}
 }
-\newline
-\\
-{Morgan~Taschuk}\textsuperscript{1,\ddag *},
-{Greg~Wilson}\textsuperscript{2,\ddag}
-\\
-\textbf{1} Ontario Institute for Cancer Research / morgan.taschuk@oicr.on.ca
-\\
-\textbf{2} Software Carpentry Foundation / gvwilson@software-carpentry.org
-\\
-\bigskip
-{\ddag} These authors contributed equally to this work.
-\\
-* Corresponding author.
 \end{flushleft}
 
 
 \begin{easylist}[checklist]
 & Use version control.
-&& Put everything created manually into version control as soon as it is created.
+&& Put everything into version control as soon as it is created.
```

Comment:
Why remove "created manually"? We don't put e.g. output files from compilation in version control.